wallets/react-wallet-v2/src/utils/HederaWalletUtil.ts

```diff
@@ -0,0 +1,38 @@
+import { HederaWallet } from '@/lib/HederaLib'
+
+export let hederaAddresses: string[] = []
+export let hederaWallet: HederaWallet
+
+export async function createOrRestoreHederaWallet() {
+  try {
+    /**
+     * !!!WARNING!!!
+     * Do not use this approach of storing/accessing private keys in production.
+     * This app is not configured with a secure storage mechanism, so this is just the
+     * best way to make it work and showcase Hedera integration with WalletConnect.
+     */
+    const privateKey = process.env.NEXT_PUBLIC_HEDERA_PRIVATE_KEY
+    const accountAddress = process.env.NEXT_PUBLIC_HEDERA_ACCOUNT_ID
+
+    if (!accountAddress || !privateKey) {
+      throw new Error(
+        'Missing required env vars: `NEXT_PUBLIC_HEDERA_ACCOUNT_ID` and/or `NEXT_PUBLIC_HEDERA_PRIVATE_KEY`'
```

Comment:
This seems to be causing an uncaught exception when attempting to toggle Testnets on under the wallets "Settings" (gear icon) tab, which then blocks the accounts/chains view from rendering:

<img width="498" alt="Screenshot 2023-08-22 at 18 38 44" src="https://github.com/WalletConnect/web-examples/assets/8663099/b8c87acc-a255-4314-a3ba-90053b1504d0">